### PR TITLE
Attach replace - fix

### PR DESCRIPTION
**Pull request by francescadecicco**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

**Comment by francescadecicco** [comment]
Screenshot 2024-03-14 at 16 57 03

File: ibm/service/globaltagging/data_source_ibm_resource_tag.go [hunk 1]
```diff
@@ -42,6 +42,13 @@ func DataSourceIBMResourceTag() *schema.Resource {
 				Description:  "Tag type on which the tags should be fetched",
 				Default:      "user",
 			},
+			"replace": {
+				Type:             schema.TypeBool,
```
Comment (hkantare) [inline ibm/service/globaltagging/data_source_ibm_resource_tag.go:46]: in datasource it shd be only computed

File: ibm/service/globaltagging/data_source_ibm_resource_tag.go [hunk 2]
```diff
@@ -60,6 +67,12 @@ func dataSourceIBMResourceTagRead(d *schema.ResourceData, meta interface{}) erro
 		tType = t.(string)
 	}
 
+	tReplace := false
```
Comment (hkantare) [inline ibm/service/globaltagging/data_source_ibm_resource_tag.go:70]: why are we reading from user input and setting back the same value.
This attribute is not required for datasource

File: ibm/service/globaltagging/resource_ibm_resource_tag.go [hunk 3]
```diff
@@ -161,6 +161,10 @@ func resourceIBMResourceTagCreate(d *schema.ResourceData, meta interface{}) erro
 			AttachTagOptions.AccountID = flex.PtrToString(accountID)
 		}
 	}
+	if v, ok := d.GetOk("replace"); ok && v != nil {
```
Comment (hkantare) [inline ibm/service/globaltagging/resource_ibm_resource_tag.go:164]: why did we add the code again already we have at line 169

**Review by hkantare — APPROVED** [review]

**Pull request merged**